debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/DecoderDifferences.java

```diff
@@ -44,4 +44,13 @@ private static boolean wal2Json() {
         return TestHelper.decoderPlugin() == PostgresConnectorConfig.LogicalDecoder.WAL2JSON || TestHelper.decoderPlugin() == PostgresConnectorConfig.LogicalDecoder.WAL2JSON_RDS;
     }
 
+    /**
+     * wal2json plugin is not currently able to encode and parse NaN and Inf values
```

Comment:
That seems worth opening an issue with the wal2json folks. We also should document it in our list of differences.

Reply:
It is not a bug but a feature :-( they are filtering it out intentionally
https://github.com/eulerto/wal2json/blob/master/wal2json.c#L649-L655

So I guess it is going to go into our docs only